modules/ossm-security-auth-policy.adoc

```diff
@@ -0,0 +1,180 @@
+////
+Module included in the following assemblies:
+-service_mesh/v2x/ossm-security.adoc
+////
+
+[id="ossm-vs-istio_{context}"]
+= Role Based Access Control (RBAC)
```

Comment:
I agree that headers need to focus on user tasks rather than the underlying features. I believe that's what these headers do. RBAC is not the object used to perform the task, even though it was the name of the old object. The examples are use cases that customers will be familiar with.

I ran it by engineering today and they agreed that [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) is a concept that our customers will understand.

Applying security to users and projects is too vague for this content.

I was a bit conflicted about whether this was a concept or a task, or whether I should add a subheading before the  I guess it is a task, so I've added a gerund.